# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xscreensaver/glx_qd-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./hacks/glx/cage.c.orig	Sun Jan  3 20:25:20 1999
+++ ./hacks/glx/cage.c	Thu Oct 21 08:44:35 1999
@@ -94,6 +94,8 @@
 
 #ifdef USE_GL
 
+#define APIENTRY
+#define CALLBACK
 
 #include <GL/glu.h>
 #include "e_textures.h"
--- ./hacks/glx/moebius.c.orig	Sun Jan  3 20:26:57 1999
+++ ./hacks/glx/moebius.c	Thu Oct 21 08:59:42 1999
@@ -94,6 +94,8 @@
 
 #ifdef USE_GL
 
+#define APIENTRY
+#define CALLBACK
 
 #include <GL/glu.h>
 #include "e_textures.h"
--- ./hacks/glx/pipes.c.orig	Thu Oct 21 09:00:14 1999
+++ ./hacks/glx/pipes.c	Thu Oct 21 09:00:25 1999
@@ -69,6 +69,9 @@
 
 #ifdef USE_GL
 
+#define APIENTRY
+#define CALLBACK
+
 #include <GL/glu.h>
 #include "buildlwo.h"
 
--- ./hacks/glx/buildlwo.c.orig	Thu Oct 21 09:00:50 1999
+++ ./hacks/glx/buildlwo.c	Thu Oct 21 09:01:01 1999
@@ -23,6 +23,9 @@
 
 #ifdef USE_GL
 
+#define APIENTRY
+#define CALLBACK
+
 #ifdef STANDALONE
 #include <GL/glx.h>
 #endif
--- ./hacks/glx/sproingies.c.orig	Thu Oct 21 09:01:27 1999
+++ ./hacks/glx/sproingies.c	Thu Oct 21 09:01:43 1999
@@ -33,6 +33,9 @@
 
 #ifdef USE_GL
 
+#define APIENTRY
+#define CALLBACK
+
 #include <GL/gl.h>
 #include <GL/glu.h>
 #include "buildlwo.h"
--- ./hacks/glx/sproingiewrap.c.orig	Thu Oct 21 09:01:57 1999
+++ ./hacks/glx/sproingiewrap.c	Thu Oct 21 09:02:17 1999
@@ -84,6 +84,9 @@
 
 #define MINSIZE 32
 
+#define APIENTRY
+#define CALLBACK
+
 #include <GL/glu.h>
 #include <time.h>
 
--- ./hacks/glx/stairs.c.orig	Thu Oct 21 09:02:42 1999
+++ ./hacks/glx/stairs.c	Thu Oct 21 09:02:55 1999
@@ -76,6 +76,9 @@
 
 #ifdef USE_GL
 
+#define APIENTRY
+#define CALLBACK
+
 #include <GL/glu.h>
 #include "e_textures.h"
 
--- ./hacks/glx/superquadrics.c.orig	Thu Oct 21 09:03:11 1999
+++ ./hacks/glx/superquadrics.c	Thu Oct 21 09:03:26 1999
@@ -130,6 +130,9 @@
 
 #endif
 
+#define APIENTRY
+#define CALLBACK
+
 #include <GL/glu.h>
 
 #define MaxRes          50
--- ./hacks/glx/atlantis.c.orig	Thu Oct 21 09:03:56 1999
+++ ./hacks/glx/atlantis.c	Thu Oct 21 09:04:11 1999
@@ -116,6 +116,9 @@
 
 #ifdef USE_GL
 
+#define APIENTRY
+#define CALLBACK
+
 #include "atlantis.h"
 #include <GL/glu.h>
 
--- ./hacks/glx/bubble3d.h.orig	Thu Oct 21 09:05:49 1999
+++ ./hacks/glx/bubble3d.h	Thu Oct 21 09:06:02 1999
@@ -12,6 +12,9 @@
 # include "xlock.h"		/* from the xlockmore distribution */
 #endif /* !STANDALONE */
 
+#define APIENTRY
+#define CALLBACK
+
 #include <GL/gl.h>
 #include <GL/glu.h>
 
--- ./hacks/glx/glplanet.c.orig	Thu Oct 21 09:06:32 1999
+++ ./hacks/glx/glplanet.c	Thu Oct 21 09:06:48 1999
@@ -93,6 +93,8 @@
 # endif /* VMS */
 #endif
 
+#define APIENTRY
+#define CALLBACK
 
 #include <GL/glu.h>
 
--- ./hacks/glx/pulsar.c.orig	Thu Oct 21 09:07:10 1999
+++ ./hacks/glx/pulsar.c	Thu Oct 21 09:07:26 1999
@@ -87,6 +87,8 @@
 # endif /* VMS */
 #endif
 
+#define APIENTRY
+#define CALLBACK
 
 #include <GL/gl.h>
 #include <GL/glu.h>
--- ./hacks/glx/sierpinski3d.c.orig	Sat Nov 13 22:24:17 1999
+++ ./hacks/glx/sierpinski3d.c	Fri Nov 26 07:38:10 1999
@@ -81,6 +81,9 @@
 
 static gasketstruct *gasket = NULL;
 
+#define APIENTRY
+#define CALLBACK
+
 #include <GL/glu.h>
 
 /* static GLuint limit; */
